src/react-components/profile-entry-panel.js

```diff
@@ -105,57 +176,35 @@ class ProfileEntryPanel extends Component {
               title={formatMessage({ id: "profile.display_name.validation_warning" })}
               ref={inp => (this.nameInput = inp)}
             />
-            {this.state.customMode ? (
-              <div className={styles.avatarSelectorContainer}>
-                <label htmlFor="#custom-avatar-url">Avatar GLTF/GLB </label>
-                <input
-                  id="custom-avatar-url"
-                  type="url"
-                  className={styles.formFieldText}
-                  value={this.state.avatarId}
-                  onChange={e => this.setState({ avatarId: e.target.value })}
-                />
-                <div className={styles.links}>
-                  <a onClick={() => this.setState({ customMode: false, avatarId: "botdefault" })}>cancel</a>
-                </div>
-              </div>
-            ) : (
-              <div className={styles.avatarSelectorContainer}>
-                <div className="loading-panel">
-                  <div className="loader-wrap">
-                    <div className="loader">
-                      <div className="loader-center" />
-                    </div>
-                  </div>
-                </div>
-                <iframe
-                  className={styles.avatarSelector}
-                  src={`/avatar-selector.html`}
-                  ref={ifr => {
-                    if (this.avatarSelector === ifr) return;
-
-                    this.avatarSelector = ifr;
-
-                    if (this.avatarSelector) {
-                      this.avatarSelector.onload = () => {
-                        this.sendAvatarStateToIframe();
-                      };
-                    }
-                  }}
-                />
+
+            <div className={styles.tabs}>
+              <a
+                onClick={() => this.setState({ avatarType: "legacy", avatarId: "botdefault" })}
+                className={classNames({ selected: this.state.avatarType === "legacy" })}
+              >
+                Default
+              </a>
+              {(this.props.customSkinEnabled || this.state.avatarType === "skinnable") && (
                 <a
-                  className="custom-url-link"
-                  onClick={() =>
-                    this.setState({ customMode: true, avatarId: avatars.find(a => a.id === this.state.avatarId).model })
-                  }
+                  onClick={() => this.setState({ avatarType: "skinnable", avatarId: null })}
+                  className={classNames({ selected: this.state.avatarType === "skinnable" })}
                 >
-                  custom url
+                  Custom Skin
                 </a>
-              </div>
+              )}
+              <a
+                onClick={() => this.setState({ avatarType: "url", avatarId: "" })}
```

Comment:
does `null` not work here? i tried it locally and it at least initialized OK

Reply:
React complains if you try to set a text input to null since the `.value` of ""